src/scala/microsoft-spark-3.0.x/src/main/scala/org/apache/spark/sql/api/dotnet/SQLUtils.scala

```diff
@@ -37,4 +43,13 @@ object SQLUtils {
       broadcastVars,
       accumulator)
   }
+
+  def createDataFrameHelper(
```

Comment:
`createDataFrame` should be good enough. Also, add a short comment similar to `createPythonFunction` above.